docker/elasticsearch/data-process-index-config.json

```diff
@@ -0,0 +1,230 @@
+{
```

Comment:
You will need to create this index at boot-up? Check this: https://github.com/linkedin/datahub/blob/master/docker/elasticsearch/Dockerfile

Reply:
This indexing feature has been planned  in the next PR.